src/rust/cryptography-x509-validation/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "cryptography-x509-validation"
+version = "0.1.0"
+authors = ["The cryptography developers <cryptography-dev@python.org>"]
+edition = "2021"
+publish = false
+# This specifies the MSRV
+rust-version = "1.56.0"
+
+[dependencies]
+asn1 = { version = "0.15.0", default-features = false }
+chrono = "0.4"
```

Comment:
Chrono is a very heavy dependency, we previously worked to remove it, and I think we'd like to keep it that way.

Reply:
Thinking about it some more, I believe we can do this by only using `asn1::DateTime` in the core Rust bits, and pushing the optionality API (i.e., "default to now if not supplied") up to the pyO3 layer.

That would allow us to remove the `chrono` dep in exchange for a slightly less flexible Rust-level API, if that seems reasonable to you (it does to me).

Reply:
https://github.com/pyca/cryptography/pull/8873/commits/9979a00f655b5648f6413972fcbb1ec5101954c1